src/store/message-store-level.ts

```diff
@@ -73,7 +73,7 @@ export class MessageStoreLevel implements MessageStore {
       return undefined;
     }
 
-    const decodedBlock = await abortOr<block.Block>(options?.signal, block.decode({ bytes, codec: cbor, hasher: sha256 }));
```

Comment:
I think usages of `block` are all removed, but we're still importing. mind removing?

While on the subject, what do you think of[ adding a plugin](https://www.npmjs.com/package/eslint-plugin-unused-imports) to the linter to catch unused imports? I can file an issue and add a PR.

Reply:
I think it does get used, `block.decode()` in same line. Also we already have a linter rule `@typescript-eslint/no-unused-vars` includes this check, I get nagged all the time when imports are not used.